src/icon.ts

```diff
@@ -7,8 +7,8 @@ enum StyleName {
 
 interface Styles {
   beforeLayer?: string;
-  imageLayout?: MapboxValues;
-  textColor?: MapboxValues;
+  imageLayout?: MapboxExpr;
+  textColor?: MapboxExpr;
```

Comment:
This is wrong, `textColor` is not an array, and it was not a `MapboxValues` actually... So just a `string`?

Reply:
`imageLayout` and `textColor` should `MapboxValues`, not `MapboxExpr`.
Fixed.